iosApp/iosApp/CountryItemCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  CountryItemCell.swift
+//  iosApp
+//
+//  Created by Dmytro Kovalenko on 03.02.2021.
+//  Copyright © 2021 orgName. All rights reserved.
+//
+
+import UIKit
+import shared
+
+class CountryItemCell: UITableViewCell {
+    
+    
+    @IBOutlet weak var countryNameLabel: UILabel!
+    @IBOutlet weak var totalCasesLabel: UILabel!
+    @IBOutlet weak var newCasesLabel: UILabel!
+    @IBOutlet weak var totalDeathsLabel: UILabel!
+    @IBOutlet weak var newDeathsLabel: UILabel!
+    
+    func setItem(item: CountryRow) {
```

Comment:
Minor, but I'd go for something like `func configureForCountry(_ country CountryRow)` instead.

Reply:
fixed